app/app/routing/schedule.py

```diff
@@ -92,25 +108,18 @@ def _shipments(self):
             } for booking in self.booking_list]
 
     @property
-    def _stations(self):
-        return {
-            627042770: {
-                "longitude": 13.3568719,
-                "latitude": 52.4815767
-            },
-            27785378: {
-                "longitude": 13.3587658,
-                "latitude": 52.4857809
-            },
-            2493824077: {
-                "longitude": 13.3498653,
-                "latitude": 52.4794034
-            }
-        }
+    def _stops(self):
+        return [627042770, 27785378, 2493824077]
```

Comment:
@wolfhardfehre the computation is actually really fast so we could even allow ambiguous start and stop positions of users and compute the matrix on the fly with a little adaptation. I tested with 20 locations and it was still a fraction of a second.